### ISSUE TITLE
Unexpected keyword argument 'disable' in version 0.1.2.0

### ISSUE THREAD

**Issue by alansill** [davidbrochart/jpterm#53]
Progress over #49 but even the latest release (verified below) installed via "pip install jpterm==0.1.2.0" is showing a problem with the disable argument:

```
$ pip show jpterm
Name: jpterm
Version: 0.1.2
Summary: Jupyter in the terminal
Home-page: 
Author: 
Author-email: David Brochart <david.brochart@gmail.com>
License: MIT
Location: /home/asill/conda/envs/jpterm/lib/python3.12/site-packages
Requires: rich-click, txl-editors, txl-file-browser, txl-jpterm, txl-local-contents, txl-local-terminals, txl-notebook-viewer, txl-remote-contents, txl-remote-terminals, txl-terminal, txl-text-viewer
Required-by: 
```

```
$ jpterm
INFO:asphalt.core.runner:Running in development mode
INFO:asphalt.core.runner:Application stopped
Traceback (most recent call last):
  File "/home/asill/conda/envs/jpterm/bin/jpterm", line 8, in <module>
    sys.exit(main())
             ^^^^^^
  File "/home/asill/conda/envs/jpterm/lib/python3.12/site-packages/jpterm/cli.py", line 31, in main
    command()
  File "/home/asill/conda/envs/jpterm/lib/python3.12/site-packages/click/core.py", line 1157, in __call__
    return self.main(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/asill/conda/envs/jpterm/lib/python3.12/site-packages/rich_click/rich_command.py", line 126, in main
    rv = self.invoke(ctx)
         ^^^^^^^^^^^^^^^^
  File "/home/asill/conda/envs/jpterm/lib/python3.12/site-packages/click/core.py", line 1434, in invoke
    return ctx.invoke(self.callback, **ctx.params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/asill/conda/envs/jpterm/lib/python3.12/site-packages/click/core.py", line 783, in invoke
    return __callback(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/asill/conda/envs/jpterm/lib/python3.12/site-packages/txl/cli.py", line 36, in txl_main
    run(kwargs)
  File "/home/asill/conda/envs/jpterm/lib/python3.12/site-packages/txl/app.py", line 30, in run
    asphalt_run.callback(unsafe=False, loop=None, service=None, **kwargs)
  File "/home/asill/conda/envs/jpterm/lib/python3.12/site-packages/asphalt/core/cli.py", line 149, in run
    run_application(**config)
  File "/home/asill/conda/envs/jpterm/lib/python3.12/site-packages/asphalt/core/runner.py", line 98, in run_application
    component = cast(Component, component_types.create_object(**component))
                                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/asill/conda/envs/jpterm/lib/python3.12/site-packages/asphalt/core/utils.py", line 190, in create_object
    return plugin_class(**constructor_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: AppComponent.__init__() got an unexpected keyword argument 'disable'
```

**Comment by alansill** [comment]
Closing as I see there is a 0.1.2.1 now - thanks.

**Comment by davidbrochart** [comment]
Note that jpterm doesn't support python 3.12 yet.

**Comment by alansill** [comment]
Reopening as it doesn't look like this will be fixed by 0.1.21 (still waiting for this release to propagate to pipy.

**Comment by alansill** [comment]
Thanks! With python 3.11.6 and jpterm 0.1.21 it works.
<img width="723" alt="Screenshot 2023-10-28 at 4 58 58 PM" src="https://github.com/davidbrochart/jpterm/assets/2364484/36732eaa-ba2e-4180-9346-0662c017a999">
